src/lsdb/catalog/dataset/healpix_dataset.py

```diff
@@ -222,8 +224,10 @@ def apply_func(df, *args, partition_info=None, **kwargs):
         else:
             output_ddf = self._ddf.map_partitions(func, *args, meta=meta, **kwargs)
```

Comment:
Minor: it's a little weird to still have variables named `*_ddf`, when their type isn't *exactly* dask data frame.

Reply:
I've been trying to normalize `ndf` as the nested-dask equivalent verbage

Reply:
Replaced for local variables, but keeping `_ddf` the same as we discussed